.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### Issues
+
+What issues are being solved?
+
+### Todos
+
+What's outstanding?
+
+### Impacted Areas
+
+What components will be affected?
+
+### Steps to Reproduce
+
+What do we need to run, or what do we need to do to test this?
+
```

Comment:
@stewartyu More of the edge case when a PR is being opened and there isn't an `Issue` for it.